com.unicornsonlsd.Finamp.yaml

```diff
@@ -0,0 +1,291 @@
+app-id: com.unicornsonlsd.Finamp
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: finamp
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --share=network
+  - --own-name=org.mpris.MediaPlayer2.*
```

Comment:
This does not seem right. It should be `org.mpris.MediaPlayer2.appname.*` if you are a MPRIS client then you need `--talk-name=org.mpris.MediaPlayer2.*`

Reply:
I guess this is an upstream issue but the app doesn't work if I make that change:

```
[Startup/INFO] 2024-05-30 18:28:57.857539: App starting, logging initialized.
[ErrorApp/SEVERE] 2024-05-30 18:28:57.887753: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
		#0      DBusClient._callMethod (package:dbus/src/dbus_client.dart:1120)
		<asynchronous suspension>
		#1      DBusClient.callMethod (package:dbus/src/dbus_client.dart:621)
		<asynchronous suspension>
		#2      DBusClient.requestName (package:dbus/src/dbus_client.dart:345)
		<asynchronous suspension>
		#3      AudioServiceMpris.configure (package:audio_service_mpris/audio_service_mpris.dart:82)
		<asynchronous suspension>
		#4      AudioService.init (package:audio_service/audio_service.dart:1007)
		<asynchronous suspension>
		#5      _setupPlaybackServices (package:finamp/main.dart:272)
		<asynchronous suspension>
		#6      main (package:finamp/main.dart:83)
		<asynchronous suspension>
```

Reply:
So it doesn't work with `--talk-name=org.mpris.MediaPlayer2.*` either? Can you please point me to the mpris code/capabilities used by the app?

Reply:
Does `--own-name=org.mpris.MediaPlayer2.appname.*` and `--talk-name=org.mpris.MediaPlayer2.*` work? Make sure to replace `appname` with the actual name used there.

Reply:
Finamp uses https://github.com/bdrazhzhov/audio-service-mpris, which hooks into https://github.com/ryanheise/audio_service/tree/minor/audio_service to transform the events generated by `audio_service` to mpris/dbus events, and vice-versa.  
Aside from adding the package, we don't use it explicitly, it all happens automatically through an interface.

Reply:
That gives me `[ErrorApp/SEVERE] 2024-05-31 09:13:24.117865: org.freedesktop.DBus.Error.AccessDenied: org.freedesktop.DBus.Error.AccessDenied` rather than `[ErrorApp/SEVERE] 2024-05-30 18:28:57.887753: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown` but otherwise the same.

Reply:
Ok I'll try it once a build gets going.